packages/realm/src/runtime.d.ts

```diff
@@ -41,3 +41,13 @@ declare const console: Console;
 declare module "buffer" {
   type Buffer = Uint8Array;
 }
+
+/** @deprecated React Native doesn't provide this */
```

Comment:
Would it make sense to add that we should use `binding.WeakRef` if needed? Or is it not the right place for it?

Reply:
It could make sense 👍 
```suggestion
/** @deprecated React Native doesn't provide this, use our `binding.WeakRef` polyfill instead. */
```